cmd/modbus-cli/main.go

```diff
@@ -274,6 +274,8 @@ func convertToBytes(eType string, order binary.ByteOrder, forcedOrder string, va
 		buf = w.ToFloat32(float32(val))
 	case "float64":
 		buf = w.ToFloat64(float64(val))
+	default:
+		err = fmt.Errorf("unsupported conversion type: '%s'", eType)
```

Comment:
Let's try to keep the error message consistent with the other errors above:

```suggestion
		err = fmt.Errorf("unsupported conversion type %s", eType)
```